handler.js

```diff
@@ -51,7 +60,28 @@ module.exports.renderFromS3 = async (event, context, callback) => {
   const replayJSON = fs.readFileSync(tmpPath);
   fs.unlinkSync(tmpPath);
   const replay = JSON.parse(replayJSON);
-  await renderVideo(replay, callback);
+  await renderVideo(replay, callback, srcKey);
+};
+
+/**
+ * Returns a URL to upload an animation JSON file to via a PUT request
+ */
+module.exports.getS3UploadURL = async (event, context, callback) => {
+  const uuid = uuidv4();
+  const expirationInSeconds = 60 * 10;
+  const url = s3.getSignedUrl('putObject', {
+    Bucket: SOURCE_BUCKET,
+    Key: uuid,
+    Expires: expirationInSeconds
+  });
+  const response = {
+    statusCode: 200,
+    body: JSON.stringify({
+      uploadURL: url,
+      resultLocation: getOutputURL(uuid)
```

Comment:
is it intentional that the url we return for a JSON animation file is a `.mp4`?

Reply:
From my understanding, in this response `uploadURL` is the URL the client will PUT its JSON animation to, then `resultLocation` is the URL the client will poll against for the mp4 results of the processed video output (which will be written after the s3-triggered event renders the video from the json source).

Reply:
Oh! I definitely misunderstood. Thanks!